### PR TITLE
feat: selectable app theme [WPB-4483]

### PR DESCRIPTION
**Pull request by Garzas**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4483" title="WPB-4483" target="_blank">StoryWPB-4483</a>  [android] Change app theme/appearance
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Implemented screen for controlling light and dark mode in app

https://github.com/wireapp/wire-android-reloaded/assets/13151239/079612e1-e46b-42a4-bcb3-47683b8097fc

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1563](https://10.10.124.134/job/AR-build-pipeline/1563/) **failed**.

**Comment by AndroidBob** [comment]
Build [1564](https://10.10.124.134/job/AR-build-pipeline/1564/) **failed**.

**Comment by AndroidBob** [comment]
Build [1565](https://10.10.124.134/job/AR-build-pipeline/1565/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/settings/appearance/AppearanceScreen.kt [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                }
+            },
+            style = MaterialTheme.wireTypography.body01,
+            color = MaterialTheme.wireColorScheme.onBackground,
+            modifier = Modifier.padding(vertical = dimensions().spacing16x)
+        )
+    }
+}
+
+@Preview(showBackground = false)
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/home/settings/appearance/AppearanceScreen.kt]: suggestion: `@PreviewMultipleThemes`

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [1592](https://10.10.124.134/job/AR-build-pipeline/1592/) **failed**.

**Comment by AndroidBob** [comment]
Build [1593](https://10.10.124.134/job/AR-build-pipeline/1593/) **failed**.

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [1599](https://10.10.124.134/job/AR-build-pipeline/1599/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7978255)(7978255)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7978255)(7978255)-staging-compat.apk)

**Review by vitorhugods — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [1612](https://10.10.124.134/job/AR-build-pipeline/1612/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.5.0(7987031)(7987031)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.5.0(7987031)(7987031)-staging-compat.apk)

**Pull request merged**